librz/include/sflib/freebsd-x86-32/sflib.h

```diff
@@ -0,0 +1,290 @@
+/*
+ * sflib.h --- SFLib syscall library for FreeBSD/i386 
+ * see http://www.secdev.org/projects/shellforge.html for more informations
+ *
+ * Copyright (C) 2004  Philippe Biondi <phil@secdev.org>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+
+
+/*
+ * Automatically generated by gensflib.py 
+ * Wed Mar 31 17:50:12 2021
+ */
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2004  Philippe Biondi <phil@secdev.org>
// SPDX-License-Identifier: LGPL-3.0-only

/*
 * sflib.h --- SFLib syscall library for FreeBSD/i386 
 * see http://www.secdev.org/projects/shellforge.html for more informations
 *
 * Copyright (C) 2004  Philippe Biondi <phil@secdev.org>
 *
 * This program is free software; you can redistribute it and/or modify it
 * under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation.
 *
 * This program is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Lesser General Public License for more details.
 */


/*
 * Automatically generated by gensflib.py 
 * Wed Mar 31 17:50:12 2021
 */
```